lib/xandra/cluster/control_connection.ex

```diff
@@ -396,6 +397,7 @@ defmodule Xandra.Cluster.ControlConnection do
 
     case transport.connect(address, port, data.transport_options, @default_timeout) do
       {:ok, socket} ->
+        Logger.debug("Connected to #{inspect(node)}")
```

Comment:
We already have the information about the node in the `xandra_address` and `xandra_port` metadata:

```suggestion
        Logger.debug("Connected")
```